index.bs

```diff
@@ -193,41 +190,32 @@ partial interface XRSession {
 
-NOTE: It is not possible for the user to perform computer vision based on automatic composition. Future modules may enable such functionality.
+- When performing <dfn>additive environment blending</dfn>, the rendered buffers obtained by the [=XR Compositor=] are composited using [=lighter=] blending before being presented on the [=XR device=]. This technique MUST be applied on [=additive light=] displays, regardless of the [=XRSession/mode=].
 
-ISSUE: Should the preceding paragraph move to the security section?
+The [=XR Compositor=] MAY make additional color or pixel adjustments to optimize the experience. The timing of composition MUST NOT depend on the [=blend technique=] or source of the [=real-world environment=]. but MUST NOT perform occlusion based on pixel depth relative to real-world geometry; only rendered content MUST be composed on top of the real-world background.
 
-Automatic composition MAY make additional color or pixel adjustments to optimize the experience but MUST NOT perform occlusion based on pixel depth relative to real-world geometry; only rendered content MUST be composed on top of the real-world background.
+NOTE: Future modules may enable automatic or manual pixel occlusion with the [=real-world environment=].
 
-NOTE: Future modules, such as those that may provide access to real world geometry data, might extend this behavior to allow for automatic pixel occlusion or possibly to enable developers to perform this occlusion manually.
+The [=XR Compositor=] MUST NOT automatically grant the page access to any additional information such as camera intrinsics, media streams, real-world geometry, etc.
 
-During automatic composition the [=xr device=] MUST have the capability of producing an [=aligned real-world background=] for each [=view=].
-An [=xr device=] MUST capture the <dfn>aligned real-world background</dfn> in such a way that it can be aligned with the pixels of the {{XRRenderState/baseLayer}}.
+NOTE: Developers may request access to an [=XR Device=]'s camera, should one be exposed through the existing WebRTC specification. However, doing so does not provide a mechanism to query the {{XRRigidTransform}} between the camera's location and the [=native origin=] of the [=viewer reference space=]. It also does not provide a guaranteed way to determine the camera intrinsics necessary to match the view of the [=real-world environment=]. As such, performing effective computer vision algorithms wil be significantly hampered. Future modules or specifications may enable such functionality.
```

Comment:
[getUserMedia](https://w3c.github.io/mediacapture-main/#dom-mediadevices-getusermedia) is how a page gets access to the camera, not WebRTC